shared/rspec/support/retry.rb

```diff
@@ -6,8 +6,8 @@
   # show exception that triggers a retry if verbose_retry is set to true
   config.display_try_failure_messages = true
 
-  # run retry only on features tests
-  config.around :each, :js do |example|
-    example.run_with_retry retry: 3
+  # run retry only on systems tests
+  config.around :each, type: :system do |example|
+    example.run_with_retry retry: ENV.fetch('TEST_RETRY').to_i
```

Comment:
@anduonghien personally i dont think this fits to be a `ENV ` variable. It could very well have been a constant like `RETRY_COUNT` inside the same file, at max.

Reply:
@abhinavmsra I recommend An to use an env variable as it can be switched on/of without modifying any file. Think that this will be enabled on CI only

Reply:
@olivierobert its something we rarely tweek